yt_dlp/extractor/facebook.py

```diff
@@ -571,16 +571,21 @@ def process_formats(info):
                 # Formats larger than ~500MB will return error 403 unless chunk size is regulated
                 f.setdefault('downloader_options', {})['http_chunk_size'] = 250 << 20
 
-        def extract_relay_data(_filter):
-            return self._parse_json(self._search_regex(
-                rf'data-sjs>({{.*?{_filter}.*?}})</script>',
-                webpage, 'replay data', default='{}'), video_id, fatal=False) or {}
+        def yield_all_relay_data(_filter):
+            for relay_data in re.findall(rf'data-sjs>({{.*?{_filter}.*?}})</script>', webpage):
+                yield self._parse_json(relay_data, video_id, fatal=False) or {}
 
-        def extract_relay_prefetched_data(_filter):
-            return traverse_obj(extract_relay_data(_filter), (
-                'require', (None, (..., ..., ..., '__bbox', 'require')),
+        def extract_relay_data(_filter):
+            return next(yield_all_relay_data(_filter), {})
```

Comment:
Shouldnt we try to find one that has info? or am i missing something?
```suggestion
            return next(filter(None, yield_all_relay_data(_filter)), {})
```

Reply:
`extract_relay_data` is used elsewhere in this method and I didn't want to change what it did